e2e/helpers/docker_compose.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shlex
+import subprocess
+from time import sleep
+
+
+def stop():
+    subprocess.check_call("docker-compose stop -t 0".split(' '))
```

Comment:
Does python have something equivalent or comparable to Ruby's [word array literal notation](http://stackoverflow.com/a/1275147/674064) (`%w(every word is an array item of type string)`)? That'd come in handy here.